packages/launch/src/focused-launch/summary/index.tsx

```diff
@@ -574,10 +574,10 @@ const Summary: React.FunctionComponent = () => {
 	// step to the user when in this launch flow.
 	// Allow changing site title when it's the default value or when it's an empty string.
 	React.useEffect( () => {
-		if ( ! isSiteTitleStepVisible && ! isValidTitle ) {
+		if ( title !== undefined && ! isSiteTitleStepVisible && isDefaultTitle ) {
```

Comment:
Should we embed the `title !== undefined` check into the computation of `isValidTitle` ?

After all, a title is valid when it's not undefined (plus, we're already checking against empty strings)

Reply:
As we discussed, `isValidTitle` is not best suited here so we'll keep using it just to enable next step and Launch button when title is defined, non-empty and, for EN, different than default title.